lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -232,24 +234,24 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
         #     # of mongo queries,
         #     # of xblocks
         # )
-        ('no_overrides', 1, True, False): (47, 1, 6, 13),
-        ('no_overrides', 2, True, False): (119, 16, 6, 84),
-        ('no_overrides', 3, True, False): (399, 81, 6, 335),
-        ('ccx', 1, True, False): (47, 1, 6, 13),
-        ('ccx', 2, True, False): (119, 16, 6, 84),
-        ('ccx', 3, True, False): (399, 81, 6, 335),
-        ('ccx', 1, True, True): (47, 1, 6, 13),
-        ('ccx', 2, True, True): (119, 16, 6, 84),
-        ('ccx', 3, True, True): (399, 81, 6, 335),
-        ('no_overrides', 1, False, False): (47, 1, 6, 13),
-        ('no_overrides', 2, False, False): (119, 16, 6, 84),
-        ('no_overrides', 3, False, False): (399, 81, 6, 335),
-        ('ccx', 1, False, False): (47, 1, 6, 13),
-        ('ccx', 2, False, False): (119, 16, 6, 84),
-        ('ccx', 3, False, False): (399, 81, 6, 335),
-        ('ccx', 1, False, True): (47, 1, 6, 13),
-        ('ccx', 2, False, True): (119, 16, 6, 84),
-        ('ccx', 3, False, True): (399, 81, 6, 335),
+        ('no_overrides', 1, True, False): (34, 0, 6, 1),
+        ('no_overrides', 2, True, False): (40, 0, 6, 1),
+        ('no_overrides', 3, True, False): (50, 0, 6, 1),
+        ('ccx', 1, True, False): (34, 0, 6, 1),
+        ('ccx', 2, True, False): (40, 0, 6, 1),
+        ('ccx', 3, True, False): (50, 0, 6, 1),
+        ('ccx', 1, True, True): (47, 0, 6, 1),
+        ('ccx', 2, True, True): (40, 0, 6, 1),
+        ('ccx', 3, True, True): (50, 0, 6, 1),
+        ('no_overrides', 1, False, False): (34, 0, 6, 1),
+        ('no_overrides', 2, False, False): (40, 0, 6, 1),
+        ('no_overrides', 3, False, False): (50, 0, 6, 1),
+        ('ccx', 1, False, False): (34, 0, 6, 1),
+        ('ccx', 2, False, False): (40, 0, 6, 1),
+        ('ccx', 3, False, False): (50, 0, 6, 1),
+        ('ccx', 1, False, True): (47, 0, 6, 1),
+        ('ccx', 2, False, True): (40, 0, 6, 1),
+        ('ccx', 3, False, True): (50, 0, 6, 1),
```

Comment:
Xblocks are loaded during the collect phase and not during ongoing grading computation.
The SQL databases will continue to be accessed.

For CCX, the override now happens at the collect phase at the modulestore layer.  And not during module render.